CMakeLists.txt

```diff
@@ -438,7 +446,9 @@ add_custom_target ( uninstall
 #-----------------------------------------------------
 # Publicize installed location to other CMake projects
 #-----------------------------------------------------
-install ( EXPORT ${PACKAGE_NAME}-targets DESTINATION "${EXPORT_INSTALL_DIR}" )
+install ( EXPORT ${PACKAGE_NAME}-targets
+  NAMESPACE ${PACKAGE_NAME}::
```

Comment:
This is the main user visible changes because it changes the exported target names, *i.e.* adds a namespace:
- `jsonfortran::jsonfortran` should be used instead of `jsonfortran`
- `jsonfortran::jsonfortran-static` should be used instead of `jsonfortran-static`
This is not required in CMake (nothing really is), but used by most CMake projects as best practice